client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -80,10 +70,17 @@ const StatsInsights = ( props ) => {
 						<div className="stats__module-column">
 							<LatestPostSummary />
 							<MostPopular />
-							{ tagsList }
+							{ ! isJetpack && (
+								<StatsModule
+									path="tags-categories"
+									moduleStrings={ moduleStrings.tags }
+									statType="statsTags"
+								/>
+							) }
 							<AnnualSiteStats isWidget />
 						</div>
 						<div className="stats__module-column">
+							{ ! isJetpack && <StatShares siteId={ siteId } /> }
```

Comment:
Fun fact: we're both fixing a parity gap between Calypso and WP Admin but also introducing a new parity gap between Simple and Atomic.

Reply:
One step at a time! This a marathon ;-)!